web/templates/task/index.html.eex

```diff
@@ -0,0 +1,33 @@
+<h2>Listing tasks</h2>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Deadline</th>
+      <th>Status</th>
+      <th>Assigned to</th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for task <- @tasks do %>
+    <tr>
+      <td><%= task.name %></td>
+      <td><%= task.deadline %></td>
+      <td><%= task.status %></td>
+      <%= if task.user != nil do %>
```

Comment:
Makes sense.
I'll probably change to `if task.user_id` as well because `task.user` is filled with an ecto association, so I think the correct way is to check `task.user_id`.